packages/ember-routing/lib/system/router.js

```diff
@@ -987,6 +988,9 @@ function calculatePostTransitionState(emberRouter, leafRouteName, contexts) {
 
 function updatePaths(router) {
   let infos = router.router.currentHandlerInfos;
+
+  if (!isArray(infos) || infos.length === 0) { return; }
```

Comment:
Can we just use `Array.isArray` here? Since `infos` is always going to be a native array, we don't need `Ember.isArray`'s support for `Enumerable` and whatnot.